website/src/pages/v1/serve/features/monitoring-tracing/prometheus.mdx

```diff
@@ -4,83 +4,234 @@ description:
+| [`graphql_envelop_deprecated_field`](#graphql_envelop_deprecated_field)             |  Counter  | `deprecatedFields`      |
+| [`graphql_envelop_schema_change`](#graphql_envelop_schema_change)                   |  Counter  | `schemaChangeCount`     |
+| [`graphql_mesh_delegate_duration`](#graphql_mesh_delegate_duration)                 | Histogram | `delegation`            |
+| [`graphql_mesh_fetch_duration`](#graphql_mesh_fetch_duration)                       | Histogram | `fetchMetrics`          |
+| [`graphql_mesh_subgraph_execute_duration`](#graphql_mesh_subgraph_execute_duration) | Histogram | `subgraphExecute`       |
+| [`graphql_mesh_subgraph_execute_errors`](#graphql_mesh_subgraph_execute_errors)     |  Counter  | `subgraphExecuteErrors` |
+
+### `graphql_yoga_http_duration`
+
+This metric tracks the duration of incoming (downstream) HTTP requests. It reports the time spent to
+process each incoming request as an histogram.
+
+It is useful to track the responsiveness of your gateway. A spike in this metric could indicate a
+performance issue and that further investigation is needed.
+
+Please not that this metric is not specific to GraphQL, it tracks all incoming HTTP requests.
+
+You can use labels to have a better understanding of the requests and group them together. A common
+filter is to include only `statusCode` with `200` value and `method` with `POST` value to get
+execution time of successful GraphQL requests only.
+
+To enable this metric, set the `http` option to `true`.
+
+#### Labels
+
+This metric includes some useful labels to help you identify requests and group them together.
+
+| Label           | Enabled                        | Descirption                                                                                                                                                                                                                                                                                                                                                                             |
+| --------------- | ------------------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `method`        | Always on                      | The HTTP method used to request the gateway endpoint.<br/><br/> Since GraphQL usually only uses `POST` requests, this can be used to filter out GraphiQL related requests. <br/><br/> It can be any HTTP verb, including unallowed ones. Which means this metric can also be used to track malformed or malicious requests.                                                             |
```

Comment:
GraphQL can also use GET

Reply:
Yeah, but not by default. You think I should in some way indicate that this is valid only for the default `POST` usage ? I feel that if someone know they can use `GET` for graphql, it will understand that it doesn't apply for its setup ?